website/docs/OrganizingTestSuites.md

```diff
@@ -188,6 +191,9 @@ Or, exclude a spec file when filtering using a suite:
 wdio wdio.conf.js --suite login --exclude ./test/specs/e2e/login.js
 ```
 
+When the `--exclude` option is provided, it will override any patterns defined by the config or capability
+level's `exclude` parameter.
```

Comment:
```suggestion
When the `--exclude` option is provided, it will override any patterns defined by the config or capability level's `exclude` parameter.
```